src/ListComponent.js

```diff
@@ -7,22 +7,28 @@ import Button from "react-bootstrap/Button";
 import InputGroup from "react-bootstrap/InputGroup"; 
 import FormControl from "react-bootstrap/FormControl"; 
 import ListGroup from "react-bootstrap/ListGroup"; 
+import Calendar from 'react-calendar';
 import Popup from 'reactjs-popup';
 import 'reactjs-popup/dist/index.css';
 
 const ListComponent = (props) => { 
     const [input, setInput] = useState('');
     const [list, setList] = useState([]);
+    const [showCal, setShowCal] = useState(false);
+    const [calDate, setCalDate] = useState(new Date());
```

Comment:
This is just temporary storage of the input from calendar, date is stored per task, and this is reset once a task is added.